packages/ra-core/src/controller/saveContext/usePickSaveContext.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+import pick from 'lodash/pick';
+import { SaveContextValue } from './SaveContext';
+
+export const usePickSaveContext = <
```

Comment:
Could you document the use case for this one? It's not immediate by reading the source

Reply:
Done but all or other `usePickXXX` are undocumented